mvlearn/__init__.py

```diff
@@ -4,5 +4,6 @@
 import mvlearn.cotraining
 import mvlearn.datasets
 import mvlearn.utils
+import mvlearn.plotting
 
 __version__ = "0.0.1"
```

Comment:
from what I remember, this version becomes the version number for the thing published to PyPi, should we change to 0.0.2 or something higher before updating the package? Your call

Reply:
yeah I guess this is all part of the next version, so should do it.